### ISSUE TITLE
Categorical Target Field having "spaces" in class labels fails for PMMLpipeline

### ISSUE THREAD

**Issue by hitesh-tarani** [jpmml/sklearn2pmml#87]
Pmmlpipeline obect had 2 steps: DictVectorizer and RandomForestClassifier estimator.
When performing sklearn2pmml, it fails when a class label of dataset is `Education Loan`, but not when it's `Education_Loan`.

Traceback:
```
Mar 07, 2018 6:00:33 PM org.jpmml.sklearn.Main run
INFO: Parsing PKL..
Mar 07, 2018 6:00:34 PM org.jpmml.sklearn.Main run
INFO: Parsed PKL in 1137 ms.
Mar 07, 2018 6:00:34 PM org.jpmml.sklearn.Main run
INFO: Converting..
Mar 07, 2018 6:00:34 PM org.jpmml.sklearn.Main run
SEVERE: Failed to convert
java.lang.IllegalArgumentException: Education Loan
	at sklearn2pmml.PMMLPipeline$1.apply(PMMLPipeline.java:294)
	at sklearn2pmml.PMMLPipeline$1.apply(PMMLPipeline.java:287)
	at com.google.common.collect.Lists$TransformingRandomAccessList$1.transform(Lists.java:638)
	at com.google.common.collect.TransformedIterator.next(TransformedIterator.java:47)
	at org.jpmml.converter.PMMLUtil.addValues(PMMLUtil.java:117)
	at org.jpmml.converter.PMMLUtil.addValues(PMMLUtil.java:106)
	at org.jpmml.converter.PMMLEncoder.createDataField(PMMLEncoder.java:112)
	at sklearn2pmml.PMMLPipeline.encodePMML(PMMLPipeline.java:102)
	at org.jpmml.sklearn.Main.run(Main.java:144)
	at org.jpmml.sklearn.Main.main(Main.java:93)

Exception in thread "main" java.lang.IllegalArgumentException: Education Loan
	at sklearn2pmml.PMMLPipeline$1.apply(PMMLPipeline.java:294)
	at sklearn2pmml.PMMLPipeline$1.apply(PMMLPipeline.java:287)
	at com.google.common.collect.Lists$TransformingRandomAccessList$1.transform(Lists.java:638)
	at com.google.common.collect.TransformedIterator.next(TransformedIterator.java:47)
	at org.jpmml.converter.PMMLUtil.addValues(PMMLUtil.java:117)
	at org.jpmml.converter.PMMLUtil.addValues(PMMLUtil.java:106)
	at org.jpmml.converter.PMMLEncoder.createDataField(PMMLEncoder.java:112)
	at sklearn2pmml.PMMLPipeline.encodePMML(PMMLPipeline.java:102)
	at org.jpmml.sklearn.Main.run(Main.java:144)
	at org.jpmml.sklearn.Main.main(Main.java:93)
```
Thanks, for the help.
